docs/source/intro/installation.rst

```diff
@@ -1,16 +1,892 @@
+   $ PostgreSQL host: localhost
+   $ PostgreSQL port: 5432
+   $ AiiDA Database name: aiida_dev
+   $ AiiDA Database user: aiida
+   $ AiiDA Database password: <password>
+   $ AiiDA repository directory: /home/wagner/.aiida/repository/
+   [...]
+   Configuring a new user with email 'richard.wagner@leipzig.de'
+   $ First name: Richard
+   $ Last name: Wagner
+   $ Institution: BRUHL, LEIPZIG
+   $ The user has no password, do you want to set one? [y/N] y
+   $ Insert the new password:
+   $ Insert the new password (again):
+
+.. admonition:: Don't forget to backup your data!
+   :class: tip title-icon-tip
+
+   See the :ref:`installation backup how-to <how-to:installation:backup>`.
+
+.. _intro:install:start_daemon:
+
+Starting the daemon
+-------------------
+
+The AiiDA daemon process runs in the background and takes care of processing your submitted calculations and workflows, checking their status, retrieving their results once they are finished and storing them in the AiiDA database.
+
+The AiiDA daemon is controlled using three simple commands:
+
+ * ``verdi daemon start``: start the daemon
+ * ``verdi daemon status``: check the status of the daemon
+ * ``verdi daemon stop``: stop the daemon
+
+.. note::
+
+    While operational, the daemon logs its activity to a file in ``~/.aiida/daemon/log/`` (or, more generally, ``$AIIDA_PATH/.aiida/daemon/log``).
+    Get the latest log messages via ``verdi daemon logshow``.
+
+.. _intro:install:final_checks:
+
+Final checks
+------------
+
+Use the ``verdi status`` command to check that all services are up and running:
+
+.. code-block:: console
+
+   $ verdi status
+
+   ✓ profile:     On profile quicksetup
+   ✓ repository:  /repo/aiida_dev/quicksetup
+   ✓ postgres:    Connected to aiida@localhost:5432
+   ✓ rabbitmq:    Connected to amqp://127.0.0.1?heartbeat=600
+   ✓ daemon:      Daemon is running as PID 2809 since 2019-03-15 16:27:52
+
+In the example output, all service have a green check mark and so should be running as expected.
+
+.. admonition:: Are you in your virtual environment?
+   :class: note title-icon-troubleshoot
+
+   Remember that in order to work with AiiDA through for example the ``verdi`` command, you need to be in your virtual environment.
+   If you open a new terminal for example, be sure to activate it first, e.g. for venv:
+
+   .. code-block:: console
+
+      $ source ~/.virtualenvs/aiida/bin/activate
+
+.. admonition:: Want to manage your install?
+   :class: seealso title-icon-read-more
+
+   See the :ref:`Installation how-to <how-to:installation>`
+
+   :link-badge:`how-to:installation:configure,Configuring,ref,badge-primary text-white`
+   :link-badge:`how-to:installation:update,Updating,ref,badge-primary text-white`
+   :link-badge:`how-to:installation:backup:software,Backing-up,ref,badge-primary text-white`
+
+.. admonition:: What next?
+   :class: seealso title-icon-read-more
+
+   Try out the ::ref:`Basic Tutorial <tutorial:basic>`.
```

Comment:
```suggestion
   Try out the ::ref:`Basic Tutorial <tutorial:basic>`.
```
I think the reference has a syntax error.

Reply:
```suggestion
   Try out the :ref:`Basic Tutorial <tutorial:basic>`.
```